server/lib/configuration.js

```diff
@@ -25,6 +25,13 @@ var conf = module.exports = convict({
     doc: 'Check if the resources are under the /dist directory',
     format: Boolean
   },
+  babel: {
+    enabled: {
+      default: true,
+      doc: 'Convert ES2015 JavaScript to ES5',
```

Comment:
I'm sorry I'm confused, maybe this need be more detailed. You are setting it to `false` in prod, which sounds like you are just gonna serve ES2015 in prod?